### PR TITLE
Add litepcie minitest

### PR DESCRIPTION
**Pull request by acomodi**
This test requries a new part for the netv2 board. There is an open PR to add all available parts automatically, so, until that is merged, this minitest requires its manual addition.

In addition, this minitest uses the LiteX generated tcl to build the bitstream with Vivado. This yields an issue, such that the DSPs inference cannot be disabled during synthesis, therefore, a portion of the unknown bits results to be belonging to the DSPs.

All the other unknown bits seem to be mostly related to the ["delay"](https://github.com/SymbiFlow/prjxray/issues/1540) PIPs of the PCIE and GTP tiles.

There are a couple of other kinds of unknown bits which are still to be identified.

### CODE REVIEW

File: minitests/litex/litepcie/Makefile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+litex-install.ok:
+	pip install -r requirements.txt
+	touch litex-install.ok
+
+riscv-gcc.ok:
+	wget -qO- https://static.dev.sifive.com/dev-tools/$(RISCV_DIR).tar.gz | tar -xz
+	touch riscv-gcc.ok
+
+build/netv2/gateware/netv2.bit: litex-install.ok riscv-gcc.ok
+	env PATH=${PATH}:${PWD}/${RISCV_DIR}/bin ${VIRTUAL_ENV}/src/litex-boards/litex_boards/targets/netv2.py --with-pcie --variant a7-100 --build
```
Comment (litghost) [inline minitests/litex/litepcie/Makefile:43]: This should source `$XRAY_VIVADO_SETTINGS` so that Vivado is on the Path when this is invoked.

Comment (acomodi) [reply minitests/litex/litepcie/Makefile]: Sure, done

Comment (litghost) [reply minitests/litex/litepcie/Makefile]: The current line doesn't work.

Comment (litghost) [reply minitests/litex/litepcie/Makefile]: This works:
```
 build/netv2/gateware/netv2.bit: litex-install.ok riscv-gcc.ok
-       source ${XRAY_VIVADO_SETTINGS} /
-               env PATH=${PATH}:${PWD}/${RISCV_DIR}/bin ${VIRTUAL_ENV}/src/litex-boards/litex_boards/targets/netv2.py --with-pcie --variant a7-100 --build
+       export PATH=${PATH}:${PWD}/${RISCV_DIR}/bin && \
+               source ${XRAY_VIVADO_SETTINGS} && \
+               ${VIRTUAL_ENV}/src/litex-boards/litex_boards/targets/netv2.py --with-pcie --variant a7-100 --build
```

Comment (acomodi) [reply minitests/litex/litepcie/Makefile]: I actually put everything in a bash script, but I think it is better to have that in the Makefile

**Review by litghost — APPROVED** [review]
LGTM

**Pull request merged**